internals/testintegration/utils.go

```diff
@@ -0,0 +1,105 @@
+//go:build integration
+
+package testintegration
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+var DefaultLayerYAML string = `
+services:
+    demo-service:
+        override: replace
+        command: sleep 1000
+        startup: enabled
+`[1:]
+
+func Setup() error {
+	cmd := exec.Command("go", "build", "./cmd/pebble")
+	cmd.Dir = getRootDir()
+	return cmd.Run()
+}
+
+func getRootDir() string {
+	wd, _ := os.Getwd()
+	return filepath.Join(wd, "../../")
+}
+
+func AllKeywordsFoundInLogs(logs []string, keywords []string) (bool, []string) {
+	var notFound []string
+
+	for _, keyword := range keywords {
+		keywordFound := false
+		for _, log := range logs {
+			if strings.Contains(log, keyword) {
+				keywordFound = true
+				break
+			}
+		}
+		if !keywordFound {
+			notFound = append(notFound, keyword)
+		}
+	}
+
+	return len(notFound) == 0, notFound
```

Comment:
My go is a bit rusty 🦀 but would it make more sense to return a slice?

```suggestion
	return len(notFound) == 0, []notFound
```

Reply:
This comment is rendered invalid by the recent discussion with Ben because we are removing this. Closing now.